lib/common/alerts.c

```diff
@@ -169,18 +169,20 @@ unpack_alert_options(xmlNode *xml, pcmk__alert_t *entry, guint *max_timeout)
     if (value != NULL) {
         long long timeout_ms = crm_get_msec(value);
 
-        entry->timeout = (int) QB_MIN(timeout_ms, INT_MAX);
```

Comment:
Commit message: "The existing code casts the timeout to int before making sure it's greater than INT_MIN."

I wrote this code months ago. Currently, I have no idea what this means. There is no INT_MIN check.